comid/svn.go

```diff
@@ -228,6 +182,42 @@ func (o TaggedMinSVN) Valid() error {
 	return nil
 }
 
+// This function converts various types to uint64 for SVN.
```

Comment:
Slightly more in line with the [godoc conventions](https://go.dev/blog/godoc):

```suggestion
// convertToSVNUint64 converts various SVN types to uint64
```

Reply:
done